components/brave_rewards/resources/ui/components/grant.tsx

```diff
@@ -145,6 +145,7 @@ class Grant extends React.Component<Props, State> {
     let type
     let promoId
     let tokens = '0.0'
+    let date = new Date(grant.expiryTime).toLocaleDateString()
```

Comment:
Other places in this file `grant.expiryTime` is multiplied by `1000`???

Reply:
for single grant we already multiply them in the reducer, but for grant list we don't do this